src/DayColumn.js

```diff
@@ -219,23 +217,155 @@ let DaySlot = React.createClass({
     })
   },
 
-  _slotStyle(startSlot, endSlot, leftOffset){
+  _getStyledEvents() {
+    let { min, startAccessor, endAccessor, step } = this.props
```

Comment:
It'd also be great if you could give a high level description (in a block comment in the file) of the algorithm used as a way to help document and maintain the code.